strawberryfields/apps/qchem/__init__.py

```diff
@@ -20,6 +20,8 @@
 
     dynamics
     vibronic
+    utils
 """
 import strawberryfields.apps.qchem.dynamics
 import strawberryfields.apps.qchem.vibronic
+import strawberryfields.apps.qchem.utils
```

Comment:
Do we definitely want to make the `utils` module visible to the user?
The alternative being to do:
```suggestion
"""
import strawberryfields.apps.qchem.dynamics
import strawberryfields.apps.qchem.vibronic
from strawberryfields.apps.qchem.utils import duschinsky
```
This would allow users to go:
```python
from strawberryfields.apps import qchem

qchem.duschinsky(...)
```